webpack/scenes/RedHatRepositories/components/RepositorySetRepository.js

```diff
@@ -68,6 +68,7 @@ class RepositorySetRepository extends Component {
     return (
       <ListView.Item
         heading={`${arch} ${releasever}`}
+        style={{ borderBottom: '1px solid #efefef' }}
```

Comment:
Should we be using a class here?

Reply:
I haven't done any styling before so I'm not sure how we setup classes. I'm happy to do it though.